src/RandomEnchants.cpp

```diff
@@ -1,63 +1,139 @@
-/*
-* Converted from the original LUA script to a module for Azerothcore(Sunwell) :D
-*/
 #include "ScriptMgr.h"
-#include "Player.h"
 #include "Configuration/Config.h"
+#include "ObjectMgr.h"
 #include "Chat.h"
+#include "Player.h"
+#include "Object.h"
 
-class RandomEnchantsPlayer : public PlayerScript{
+bool RandomEnchantEnabled = true;
+bool RandomEnchantAnnounce = true;
+bool Crafted = 1;
+bool Looted = 1;
+bool QuestReward = 1;
+uint32 HighQuality = 5;
+uint32 LowQuality = 1;
+uint32 Chance1 = 60;
```

Comment:
why define them here? why not directly inline (see other comment below)

Reply:
hmm in fact they are already set to default values (60 65 70) directly below in the code

IN FACT, why do you put default values ? the defautl values can stay in the config too because if they are missing, the module will use the .conf.dist (or display error and then the user has to fix it)